develop-docs/DECISIONS.md

```diff
@@ -384,3 +384,20 @@ We can also start with this option to evaluate Swift Codable and switch to optio
 #### Cons
 
 1. Duplicate code.
+
+## Platform version support
+
+Date: March 11, 2025
+Contributors: @armcknight, @philipphofmann, @kahest
+
+We will support versions of each platform going back 4 major versions. There are 3 considerations:
+
+1. The distribution of events we receive from the various versions of iOS etc in the wild.
+1. Xcode support. As of the time of this writing, the oldest version of Xcode that can still submit apps to the app store is Xcode 15, which supports back to iOS 12, while the current is iOS 18.
```

Comment:
`m`: I that's key. You can still submit apps with Xcode 15, which still supports debugging apps on iOS 12. I think we can reasonably always bump it to the min debuggable Xcode version with which you can still submit apps to the app store. So by the end or mid of April, when you can only submit with Xcode 16, it's OK to raise this to iOS 15. For now it's still iOS 12. WDYT, @kahest and @armcknight?

Reply:
If we would ship a major now, I would bump to iOS 14 for sure, because you won't be able to submit apps to the app store with Xcode 15 soon.

Reply:
I would bump to iOS 14 even if it was possible to use Xcode 15 longer than a few months, because supporting 4 major versions back whenever we ship a major should be more than enough backwards compatibility in pretty much all cases, which is why I suggest the major-4 rule as default. do you agree?

Reply:
OK, then let's keep it simple. Make it major four, and if somebody complains, we revisit. We don't do a major that often, and clients could stick to an older version.